test/integration/test.js

```diff
@@ -15,7 +15,6 @@ const packages = new Map([
 	['np', 'https://github.com/sindresorhus/np'],
 	['ora', 'https://github.com/sindresorhus/ora'],
 	['p-map', 'https://github.com/sindresorhus/p-map'],
-	['detect-indent', 'https://github.com/sindresorhus/detect-indent'],
```

Comment:
Why remove this?

Reply:
Fixtures like this one cause parse errors when integrating test runs eslint https://github.com/sindresorhus/detect-indent/blob/master/fixture/fifty-fifty-space-first.js